data/lib/core/storages.lua

```diff
@@ -2111,7 +2117,176 @@ Storage = {
 			Braindeath = {},
 			DarashiaDragon = {},
 			HotCuisine = {},
-			KillingInTheNameOf = {},
+			KillingInTheNameOf = {
+				MonsterKillCount = {
+					KillCount = 42150,
```

Comment:
nitpicking: Shouldn't the id start as `42151` like the commented range on U8_5 or at least update the commentary?

Reply:
changed comment